### PR TITLE
Initial Docker Compose version for the Cinnamon Vizceral visualization.

### PR DESCRIPTION
**Pull request by henrikengstrom**

### CODE REVIEW

File: cinnamon/vizceral/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    networks:
+      - cinnamon
+  backend:
+    image: lightbend-docker-registry.bintray.io/cinnamon/cinnamon-vizceral:0.1
+    ports:
+      - "8999:8999"
+    networks: 
+      - cinnamon
+    links:
+      - elasticsearch
```
Comment (pvlugter) [inline cinnamon/vizceral/docker-compose.yml]: Are links needed, given the bridge network?

**Review by pvlugter — APPROVED** [review]
Tried it out with Reactive Maps and all worked fine.

File: cinnamon/vizceral/docker-compose.yml (hunk 1 above)
Comment (henrikengstrom) [reply cinnamon/vizceral/docker-compose.yml]: Links are also used as `dependencies` IIRC. Can change to use `dependencies` instead?

Comment (pvlugter) [reply cinnamon/vizceral/docker-compose.yml]: Yeah, `dependencies` could make more sense.

**Pull request merged**